### PR TITLE
all program

### PR DESCRIPTION
**Pull request by vishkii**
all program

### CODE REVIEW

File: p26.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import java.util.Scanner;
+
+/* return a "rotated left 2" version where the first 2 chars are moved to the end
+ * the string length will be at least 2. 
+ * 
+ */
+public class p26
```
Comment (krupaljpatel) [inline p26.java]: class name should always be in capital letter.

File: p3.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+	public static void main(String args[])
+	{
+			Scanner input = new Scanner(System.in);
+			int days;	
+			System.out.println("enter any number :  ");
+			days = input.nextInt();
+			convertdaysintomonthsanddays(days);
+	}
+		public static void convertdaysintomonthsanddays(int days)
```
Comment (krupaljpatel) [inline p3.java]: method names should be camel cased.

**Comment by arashiya** [comment]
Vishal, You never merge a pull request for your code. It will be merged by us.

**Comment by vishkii** [comment]
Ok.

**Pull request closed without merging**